### PR TITLE
Add node in documentation saying the multi-buildpack is now detected by default, no more need for BUILDPACK_URL

### PR DESCRIPTION
**Pull request by Soulou**
Part of fix for https://github.com/Scalingo/one-stop-shop/issues/28

### CODE REVIEW

**Comment by Soulou** [comment]
I've switched this to a draft again, waiting for actual changes to be deployed to prod to have the content of this doc modification to be true

File: _posts/platform/deployment/buildpacks/2000-01-01-multi.md [hunk 1]
```diff
@@ -11,12 +11,13 @@ You may need to combine several technologies in your project, this buildpack is
...
-By setting the configuration variable `BUILDPACK_URL`, the Scalingo's deployment stack will
-automatically fetch the given buildpack to deploy your application.
+If your application containers `.buildpacks` file at its root, our deployment
+system will automatically use this buildpack.
 
-```bash
-scalingo env-set BUILDPACK_URL=https://github.com/Scalingo/multi-buildpack.git
-```
+{% note %}
+The environment variable `BUILDPACK_URL` used to be requried to use the
```
Comment (ksol) [inline _posts/platform/deployment/buildpacks/2000-01-01-multi.md:18]:
```suggestion
The environment variable `BUILDPACK_URL` used to be required to use the
```

File: _posts/platform/deployment/buildpacks/2000-01-01-multi.md [hunk 2]
```diff
@@ -11,12 +11,13 @@ You may need to combine several technologies in your project, this buildpack is
 
 ## Set up this buildpack for your application
 
-By setting the configuration variable `BUILDPACK_URL`, the Scalingo's deployment stack will
-automatically fetch the given buildpack to deploy your application.
+If your application containers `.buildpacks` file at its root, our deployment
```
Comment (ksol) [inline _posts/platform/deployment/buildpacks/2000-01-01-multi.md:14]:
```suggestion
If your application contains a `.buildpacks` file at its root, our deployment
```

**Comment by Soulou** [comment]
Ahaha the monkey strikes back

**Comment by Soulou** [comment]
Thanks for proof reading

**Pull request merged**